src/components/containers/notification/Notification.js

```diff
@@ -110,27 +104,24 @@ function renderExtraAlert(alert) {
  * inverse that kind of goes against normal thinking
  */
 const CallsToAction = styled.div`
-  align-self: flex-end;
   display: flex;
   flex-direction: row-reverse;
-  padding: 0 15px 15px 0;
 
   & > button:not(:first-of-type) {
     margin-right: 15px;
   }
 
   @media (max-width: 767px) {
+  display: block;
+  padding: 0 15px;
+
+  & > button {
     display: block;
-    padding: 15px;
+    margin-bottom: 15px;
```

Comment:
Are the indents here alright? It's throwing me off slightly.

Reply:
No, these aren't quite right. And the buttons look weird when clicked, so I need to fix those.